testing/tests.js

```diff
@@ -1,7 +1,41 @@
-it("page has title", () => {
-  expect(document.getElementsByTagName("h1")[0].innerText).toEqual(
-    "Headlines page"
-  );
+it("fetches headline thumbnail as expected", () => {
+  mockFetch(myMockData)
+    .then((response) => response.json())
+    .then((data) =>
+      expect(data.response.content.fields.thumbnail).toEqual(
+        "https://media.guim.co.uk/e5517000b6c4c991eff6fff9e496b1af7e52595a/0_330_2448_1469/500.jpg"
+      )
+    );
```

Comment:
this is just testing your mock, not actually testing you code. In order to do that, you have to replace the real fetch with your mockFetch, and then call your functions.